statsmodels/genmod/families/family.py

```diff
@@ -374,7 +374,9 @@ def loglike(self, endog, mu, scale=1.):
         llf = scale * sum(-mu + endog*log(mu) - gammaln(endog+1))
         where gammaln is the log gamma function
         """
-        return scale * np.sum(-mu + endog*np.log(mu)-special.gammaln(endog+1))
+        loglike = np.sum(np.log(data_weights) - mu + endog * np.log(mu) -
+                         special.gammaln(endog+1))
```

Comment:
When you change lines like this, you could add spaces around all operator except power `**`  `endog + 1`